src/components/common/SEO/index.ts

```diff
@@ -0,0 +1 @@
+export * from './SEO'
```

Comment:
👌 i like this /index.ts `export *` pattern

Reply:
I think it's just an ES pattern that i've used in different projects over the years. Really like it for fuzzy searching too